cadquery/sketch.py

```diff
@@ -386,18 +387,42 @@ def distribute(
         Distribute locations along selected edges or wires.
         """
 
+        if n < 1:
+            raise ValueError(f"At least 1 element required, requested {n}")
+
         if not self._selection:
             raise ValueError("Nothing selected to distribute over")
 
-        params = [start + i * (stop - start) / n for i in range(n + 1)]
+        if 1 - abs(stop - start) < TOL:
+            trimmed = False
+        else:
+            trimmed = True
+
+        params = (
```

Comment:
Could you split `params` into two variables?

Reply:
Done.  Thanks for reviewing.